docker/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2017-2022 @polkadot/apps authors & contributors
+# This software may be modified and distributed under the terms
+# of the Apache-2.0 license. See the LICENSE file for details.
+
+# Fail fast on any non-zero exits
+set -e
+
+echo "*** Started building $APP"
+
+if [[ $NODE_ENV = "production" ]]
+then
+  echo "*** Building $NODE_ENV $APP"
+  docker build --progress=plain \
+    --no-cache \
+    --build-arg APP=$APP \
+    --build-arg PORT_NGINX=$PORT_NGINX \
+    --build-arg PUBLIC_URL=$PUBLIC_URL \
+    -t $APP \
+    -f docker/Dockerfile.prod .
+else
```

Comment:
we don't need this `else` statement since for development we are running docker-dev.sh, which uses `docker compose ...` (that already handles building the image) instead of `docker run ...`.
so here just keep the code in the `if` block and remove the `if` statement all together since only docker-prod.sh calls this file anyway